src/entity/archer.c

```diff
@@ -8,8 +8,8 @@ void archerAttack(Entity* _archer, Room* _active_room) {
     PlayerSpecData* pspec = (PlayerSpecData*)_archer->spec;
 
     Entity projectile = projectileInit(
-        (ivec2){.x = _archer->position.x >> POSITION_FIXED_SCALAR, .y = _archer->position.y >> POSITION_FIXED_SCALAR},
-         PLAYER, 0); // <- TEXTURE offset
```

Comment:
I think it is better to keep this comment. It just easy to forgot, that this 0 mean